### PR TITLE
Elevator

### PR DESCRIPTION
**Pull request by tishyac3141**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Elevator.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    backMotor.setNeutralMode(NeutralMode.Brake);
+
+    frontMotor.setInverted(false);
+    backMotor.setInverted(false);
+
+    backMotor.follow(frontMotor);
+
+  }
+
+  public void ascend(){
```
Comment (2stinkysocks) [inline src/main/java/frc/robot/subsystems/Elevator.java]: To keep Leo happy, add a space before the opening bracket on all of the methods in this class.

File: src/main/java/frc/robot/commands/AscendToTop.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {
+    elevator.stop();
+  }
+
+  // Returns true when the command should end.
+  @Override
+  public boolean isFinished() {
+    return false;
```
Comment (leomwilson) [inline src/main/java/frc/robot/commands/AscendToTop.java]: This needs to have a termination condition, probably a top limit switch.

File: src/main/java/frc/robot/commands/DescendToBottom.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {
+    elevator.stop();
+  }
+
+  // Returns true when the command should end.
+  @Override
+  public boolean isFinished() {
+    return false;
```
Comment (leomwilson) [inline src/main/java/frc/robot/commands/DescendToBottom.java]: Same here, should probably use a bottom limit switch.

**Review by leomwilson — CHANGES_REQUESTED** [review]
Other than the comments I made, looks good

**Review by leomwilson — APPROVED** [review]
All good

**Pull request merged**